docs/quickstart/web.md

```diff
@@ -0,0 +1,53 @@
+---
+bodyclass: docs
+layout: docs
+headline: Web Quickstart
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+<p class="lead">This guide gets you started with gRPC-Web with a simple
+working example from the browser.</p>
+
+<div id="toc"></div>
```

Comment:
Is this needed?

Reply:
I am not sure why it's there but it's in every `docs/quickstart/<lang>.md`. We could probably remove all of them in one separate PR.